sdk/src/main/java/com/bazaarvoice/emodb/sdk/EmoStartMojo.java

```diff
@@ -306,7 +306,7 @@ private void initializeRolesAndApiKeys() throws MojoFailureException {
                         .path(name)
                         .queryParam("APIKey", adminApiKey)
                         .type("application/x.json-create-role")
-                        .put(String.class, JsonHelper.asJson(entity));
+                        .post(String.class, JsonHelper.asJson(entity));
```

Comment:
This line isn't really related to this PR but it's a bug that was uncovered because of this PR.  Creating roles is a `POST` request, not a `PUT`.